File: FleeMan/src/main/java/com/example/demo/Entity/AirportMaster.java
package com.example.demo.Entity;

import jakarta.persistence.CascadeType;
import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.FetchType;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.JoinColumn;
import jakarta.persistence.ManyToOne;
import jakarta.persistence.Table;
import lombok.Data;

@Entity
@Table(name = "airport_master")
@Data
public class AirportMaster {
	
	@Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "airport_id")
    private int airportId;
	
	@Column(name = "airport_name")
	private String airportName;
	
	@ManyToOne(cascade = CascadeType.ALL, fetch = FetchType.EAGER)
    @JoinColumn(name = "city_id")
    private CityMaster city;
	
    @ManyToOne(cascade = CascadeType.ALL, fetch = FetchType.EAGER)
    @JoinColumn(name = "state_id")
    private StateMaster state;
    
    @ManyToOne(cascade = CascadeType.ALL, fetch = FetchType.EAGER)
    @JoinColumn(name = "hub_id")
    private HubMaster hub;
    
    @Column(name = "airport_code")
    private String airportCode;

}

File: Fleeman_Dotnet/Models/airport_master.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using Microsoft.EntityFrameworkCore;

namespace Fleeman_Dotnet.Models;

[Table("airport_master")]
[Index("hub_id", Name = "FK9q13ys5sara44e3t8iwsbehe7")]
[Index("state_id", Name = "FKgykh73g2t7b9tyhou2eve5ljf")]
[Index("city_id", Name = "FKnacm5228qoxi0egygij94kqje")]
public partial class airport_master
{
    [Key]
    public int airport_id { get; set; }

    [StringLength(255)]
    public string? airport_code { get; set; }

    [StringLength(255)]
    public string? airport_name { get; set; }

    public int? city_id { get; set; }

    public int? hub_id { get; set; }

    public int? state_id { get; set; }

    [ForeignKey("city_id")]
    [InverseProperty("airport_masters")]
    public virtual city_master? city { get; set; }

    [ForeignKey("hub_id")]
    [InverseProperty("airport_masters")]
    public virtual hub_master? hub { get; set; }

    [ForeignKey("state_id")]
    [InverseProperty("airport_masters")]
    public virtual state_master? state { get; set; }
}
